osu.Game/Screens/Select/Footer.cs

```diff
@@ -104,8 +104,8 @@ public Footer()
             updateModeLight();
         }
 
-        protected override bool OnMouseDown(MouseDownEvent e) => true;
+        protected override bool OnMouseDown(MouseDownEvent e) => false;
```

Comment:
why are these here

Reply:
I'm pretty sure those don't exist on the current diff?